tests/test_rotations.py

```diff
@@ -0,0 +1,338 @@
+        [[0, 1, 0], [0, 0, 1]],
+    ]
+
+    for t1, t2 in vectors:
+        rot = Rotation().from_basis(t1, t2)
+        t1_rot = rot * [1, 0, 0]
+        t2_rot = rot * [0, 1, 0]
+        np.testing.assert_array_less(
+            np.linalg.norm(t1 - t1_rot),
+            mpy.eps_quaternion,
+            "test_rotation_matrix: compare t1",
+        )
+        np.testing.assert_array_less(
+            np.linalg.norm(t2 - t2_rot),
+            mpy.eps_quaternion,
+            "test_rotation_matrix: compare t2",
+        )
+
+
+def test_transformation_matrix():
+    """Test that the transformation matrix is computed correctly."""
+
+    rotation_vector_large = [1.0, 2.0, np.pi / 5.0]
+    rotation_large = Rotation.from_rotation_vector(rotation_vector_large)
+    rotation_vector_small = (
+        rotation_vector_large
+        / np.linalg.norm(rotation_vector_large)
+        / 10.0
+        * mpy.eps_quaternion
+    )
+    rotation_small = Rotation.from_rotation_vector(rotation_vector_small)
+
+    # Test transformation matrix
+    transformation_matrix_large_reference = np.array(
+        [
+            [0.5959488405656389, 0.49803685445056006, -0.9422331516950085],
+            [-0.13028167626739845, 0.8717652242030102, 0.6155336966099826],
+            [1.0577668483049911, -0.3844663033900173, 0.5403060272710478],
+        ]
+    )
+    assert np.allclose(
+        rotation_large.get_transformation_matrix(),
+        transformation_matrix_large_reference,
+        atol=mpy.eps_quaternion,
+        rtol=0.0,
+    )
+    assert np.allclose(
+        rotation_small.get_transformation_matrix(),
+        np.identity(3),
+        atol=mpy.eps_quaternion,
+        rtol=0.0,
+    )
+
+    # Test transformation matrix inverse
+    transformation_matrix_inverse_large_reference = np.array(
+        [
+            [0.44154375784863675, 0.05812896596538626, 0.7037804689849043],
+            [0.4501610612860693, 0.8227612782872283, -0.15228520755418612],
+            [-0.5440964474342915, 0.4716532506554118, 0.36463746593568075],
+        ]
+    )
+    assert np.allclose(
+        rotation_large.get_transformation_matrix_inv(),
+        transformation_matrix_inverse_large_reference,
+        atol=mpy.eps_quaternion,
+        rtol=0.0,
+    )
+    assert np.allclose(
+        rotation_small.get_transformation_matrix_inv(),
+        np.identity(3),
+        atol=mpy.eps_quaternion,
+        rtol=0.0,
+    )
+
+
+def test_smallest_rotation_triad():
+    """Test that the smallest rotation triad is calculated correctly."""
+
+    # Get the triad obtained by a smallest rotation from an arbitrary triad
+    # onto an arbitrary tangent vector.
+    rot = Rotation([1, 2, 3], 0.431 * np.pi)
+    tan = [2.0, 3.0, -1.0]
+    rot_smallest = smallest_rotation(rot, tan)
+
+    rot_smallest_ref = [
+        0.853329730651268,
+        0.19771093216880734,
+        0.25192421451158936,
+        0.4114279380770031,
+    ]
+    np.testing.assert_array_less(
```

Comment:
I know this doesn't come from you, but do we want to switch all numpy array comparisons to `allclose` and provide a global standard for `rtol` and `atol`? Does not have to be part of this PR.

Reply:
Definitely - I did not even know that this option exists in numpy before. 

`allclose` makes more sense